packages/blade/.storybook/react-native/storybook.requires.js

```diff
@@ -23,10 +23,13 @@ argsEnhancers.forEach((enhancer) => addArgsEnhancer(enhancer));
 
 const getStories = () => {
   return [
+    require('../../src/components/Badge/Badge.stories.tsx'),
     require('../../src/components/Button/Button/Button.stories.tsx'),
     require('../../src/components/Checkbox/Checkbox.stories.tsx'),
     require('../../src/components/Checkbox/CheckboxGroup.stories.tsx'),
     require('../../src/components/Icons/Icons.stories.tsx'),
+    require('../../src/components/Link/BaseLink/BaseLink.stories.tsx'),
```

Comment:
When this runs on my machine it generates this file with double quotes 😬

Reply:
Yeah same for me. We just need to run prettier on it. I added it to the script now